Test/PcapDeviceTest.cs

```diff
@@ -29,6 +31,31 @@ namespace Test
     [NonParallelizable]
     public class PcapDeviceTest
     {
+        [Test]
+        public void DeviceProperties([PcapDevices] DeviceFixture fixture)
+        {
+            var device = (PcapDevice)fixture.GetDevice();
+
+            device.Open();
+            var pcapIf = device.Interface;
+
+            Assert.IsTrue(device.Opened);
+
+            Assert.IsNotEmpty(device.Name);
+            Assert.AreEqual(device.Name, pcapIf.Name);
+            Assert.AreEqual(device.Description, pcapIf.Description);
+
+            Assert.IsNotNull(pcapIf.GatewayAddresses);
```

Comment:
is the idea that all devices should have some gateway addresses?

Reply:
From the Framework Design Guidelines 2nd Edition (pg. 256):
>
    DO NOT return null values from collection properties or from methods returning collections. Return an empty collection or an empty array instead.


https://stackoverflow.com/a/1970058/1438522